### PR TITLE
#165770236  Create about page

### PR DESCRIPTION
**Pull request by vic3king**
#### Description
This PR create the About me page of  `Authors Haven`
#### Type of change

- [x] New Feature

#### How Has This Been Tested?

- [x] Unit testing

#### Checklist:
- [x] create About page component
- [x] Add styles
- [x] Write test
#### PT-ID

[#165770236 ](https://www.pivotaltracker.com/story/show/165770236)

#### Screenshots
image
image
image

### CODE REVIEW

File: src/components/presentations/Hero-view/heroView.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Akinmyde) [inline src/components/presentations/Hero-view/heroView.scss]: Nitpick, remove the space here

Comment (vic3king) [reply src/components/presentations/Hero-view/heroView.scss]: 😂Fixed.

File: src/components/presentations/AboutPage/about-page.scss [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  background-size: cover;
+}
+
+.text-container {
+  width: 555px;
+  height: 359px;
+}
+
+.text {
+  font-family: 'Roboto', sans-serif;
```
Comment (walsamlee) [inline src/components/presentations/AboutPage/about-page.scss]: This `font-family` declaration could have been done at an upper level of `css`, `index.css` for instance.

Comment (walsamlee) [reply src/components/presentations/AboutPage/about-page.scss]: The `font-family` has already been declared in `index.scss`
Screen Shot 2019-05-05 at 5 08 27 PM

Comment (vic3king) [reply src/components/presentations/AboutPage/about-page.scss]: Nice, I've changed the implementation.

**Pull request merged**